contracts/ERC721A.sol

```diff
@@ -712,6 +703,192 @@ contract ERC721A is IERC721A {
             }
     }
 
+    /**
+     * @dev Equivalent to `_batchTransferFrom(from, to, tokenIds)`.
+     */
+    function _batchTransferFrom(
+        address from,
+        address to,
+        uint256[] memory tokenIds
+    ) internal virtual {
+        _batchTransferFrom(address(0), from, to, tokenIds);
+    }
+
+    /**
+     * @dev Transfers `tokenIds` in batch from `from` to `to`.
+     *
+     * Requirements:
+     *
+     * - `from` cannot be the zero address.
+     * - `to` cannot be the zero address.
+     * - `tokenIds` tokens must be owned by `from`.
+     * - `tokenIds` must be strictly ascending.
```

Comment:
is there a benifit if the Ids are not only ascending, but also sequential? i.e. 0-4, 5-100 etc

Reply:
question answered below in code:
if (tokenId != end)
                    if (_packedOwnerships[tokenId] == uint256(0)) _packedOwnerships[tokenId] = prevOwnershipPacked;